src/main/java/algorithmic_task/Task.java

```diff
@@ -0,0 +1,74 @@
+package algorithmic_task;
+
+import org.springframework.stereotype.Component;
+
+@Component
+public class Task {
```

Comment:
What is the purpose of this class and why it is a bean?

Reply:
I created this class to have more information what I got as a result, not only "true", or "false". But now it is not necessary so I remove this class.